src/components/OpenOfficeButton.js

```diff
@@ -1,6 +1,6 @@
-import styles from '../styles/components/Button.module.css';
+import styles from '../styles/components/OpenOfficeButton.module.css';
```

Comment:
sorry i wasn't as clear last time, more like "3dButton" or "StyledButton" or smth like that. We still want the button to be reusable for the rest of the website